terraform/webhook/bigquery.py

```diff
@@ -50,8 +52,9 @@ def write_summarization_to_table(
       summary (str): the text summary of the document
       summary_uri (str): the Storage URI of the summary TXT document
       timestamp (datetime): when the processing occurred
+      credentials (Credentials): TODO(nicain)
     """
-    client = bigquery.Client()
+    client = bigquery.Client(credentials=credentials)
```

Comment:
Do we need to pass in credentials when we instantiate the BQ client? The prototype worked without this step.

Reply:
Removed.